develop/devguide/Core.InterAppCalls/InterAppCalls_GettingStarted.md

```diff
@@ -5,7 +5,7 @@ uid: InterAppCalls_GettingStarted
 # Getting Started
 
 > [!NOTE]
-> An example connector "Skyline Example InterAppCalls" is available in the [SkylineCommunications/SLC-C-Example_InterAppCalls](https://github.com/SkylineCommunications/SLC-C-Example_InterAppCalls) GitHub repository.
+> An example is available in the [SkylineCommunications/SLC-S-Example_InterAppCalls](https://github.com/SkylineCommunications/SLC-S-Example_InterAppCalls) GitHub repository.
```

Comment:
This is a solution and not a connector. We probably want to still point to the connector that shows how to use the interapp.

Reply:
I've updated it to point to all three.